tools/devtool

```diff
@@ -661,6 +667,11 @@ cmd_help() {
     echo "        Builds the development container from its Dockerfile."
     echo "        -n, --no-python-package-update  Do not update python packages."
     echo ""
+    echo "    build_kernel -c|--config [-n|--nproc]"
+    echo "        Builds a kernel version based on the config file provide as parameter."
```

Comment:
Maybe it'd help to display the fact that this is used for the Firecracker CI, not for production.

Reply:
Ok, done.